client/me/billing-history/receipt.jsx

```diff
@@ -126,6 +130,36 @@ const BillingReceipt = React.createClass( {
 		);
 	},
 
+	renderTitle() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<QueryBillingTransactions />
+				<HeaderCake backHref={ purchasesPaths.billingHistory() }>
+					{ translate( 'Billing History' ) }
+				</HeaderCake>
+			</div>
+		);
+	},
+
+	renderPlaceholder() {
+		const { translate } = this.props;
+
+		return (
+			<Main>
+				{ this.renderTitle() }
+				<Card compact className="billing-history__receipt-card">
+					<div className="billing-history__receipt">
+						<div className="billing-history__receipt-loading">
+							{ translate( 'Loading…' ) }
```

Comment:
That would work great - and I'll probably consider doing that for both billing history list and receipt pages for consistency. Adding to #10554.